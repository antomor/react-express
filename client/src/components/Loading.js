import React from 'react';
import { withStyles } from 'material-ui/styles';
import { LinearProgress } from 'material-ui/Progress';

const styles = {
  root: {
    flexGrow: 1,
  },
};

const Loading = props => {
  const { classes } = props;
  return (
    <div className={classes.root}>
      <LinearProgress />
    </div>
  );
}

export default withStyles(styles)(Loading);