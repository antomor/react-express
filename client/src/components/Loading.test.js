import React from 'react';
import Loading from './Loading';
import renderer from 'react-test-renderer';

describe('Loading', () => {
  it('render', () => {
    const component = renderer.create(
      <Loading />,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

}); 
