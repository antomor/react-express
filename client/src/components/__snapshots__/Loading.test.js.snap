// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading renders 1`] = `
<div
  className="Loading-root-1"
>
  <div
    className="MuiLinearProgress-root-2 MuiLinearProgress-secondaryColor-6"
    role="progressbar"
  >
    <div
      className="MuiLinearProgress-bar-9 MuiLinearProgress-secondaryColorBar-7 MuiLinearProgress-indeterminateBar1-14"
      style={Object {}}
    />
    <div
      className="MuiLinearProgress-bar-9 MuiLinearProgress-secondaryColorBar-7 MuiLinearProgress-indeterminateBar2-15"
      style={Object {}}
    />
  </div>
</div>
`;
